packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -58,22 +58,23 @@ const resolveSideEffects = () => ({
 /**
  * Get media from X-UA-Device header set by Fastly which parses the user agent string
  */
-const getMedia = (userAgent: string) => {
+const getMedia = (userAgent: string, userAgentDevice: string) => {
```

Comment:
are there any tests that should be updated for `getMedia`?

Reply:
no unfortunately not :(